CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+**Changes**
+
+- Arrays now have a fixed capacity
```

Comment:
Will we ever introduce variable-length arrays (vectors, arraylists or something)?

Reply:
Yes, probably as part of the standard library.

Reply:
Except that it would be ultra painful to keep JS backend at that point since implementing such structure in C vs. JS is completely different

Reply:
Not really. We can gear the language towards C. JS will support all of that in a very forgiving manner. Arrays for example technically have a fixed size in antimony now, but for JS that restriction doesn't exist until we add proper checking logic.